src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableWriter.class.php

```diff
@@ -417,6 +490,8 @@ protected function getTableEngine($postData)
      * @param string $tableName
      *
      * @return string
+     *
+     * @throws DBALException
```

Comment:
It's OK to add these declarations. Overall I'd like to ask if we should really let DBAL bleed into that much business code as it is now, but this is a separate and global topic and not really worsened by this change.